Main.java

```diff
@@ -24,9 +27,9 @@ public static void main(String[] args) {
 
         System.out.println();
 
-        System.out.println("Ticket 1 price: $" + ticket1.getPrice());
-        System.out.println("Ticket 2 price: $" + ticket2.getPrice());
-        System.out.println("Ticket 5 price: $" + ticket5.getPrice());
+        System.out.println("Walkup ticket price: $" + ticket1.getPrice());
+        System.out.println("Advance ticket price: $" + ticket2.getPrice());
+        System.out.println("Student advance ticket price: $" + ticket5.getPrice());
```

Comment:
I like this change, it makes it a lot more clear how much each type of ticket costs.